glean-core/python/glean/glean.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from . import _ffi
+
+
+class Glean:
+    """
+    The main Glean API.
+
+    This is exposed through the global `Glean` object.
+    """
+
+    _handle = 0
+
+    @classmethod
+    def initialize(cls, configuration, data_dir):
+        """
+        Initialize the Glean SDK.
+
+        This should only be initialized once by the application, and not by
+        libraries using the Glean SDK. A message is logged to error and no
+        changes are made to the state if initialize is called a more than
+        once.
+
+        Args:
+            configuration (glean.config.Configuration): An object with global
+                settings.
+            data_dir (pathlib.Path): The path to the Glean data directory.
+        """
+        if cls.is_initialized():
+            return
+
+        cls._configuration = configuration
+        cls._data_dir = data_dir
+
+        cfg = _ffi.make_config(cls._data_dir, "glean")
+
+        cls._handle = _ffi.lib.glean_initialize(cfg)
+
+        if cls._handle == 0:
```

Comment:
In Kotlin we [silently fail](https://github.com/mozilla/glean/blob/88501e0292fb32da4bdc68dbe2c12a4291fc7eb0/glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt#L163). Should we keep the same behaviour here?

Reply:
We probably should, but I just wanted to make it easy write a basic unit test to ensure that the FFI is working.  This whole function is basically a placeholder for what it will really be.